data/navigations_de.yml

```
@@ -0,0 +1,98 @@
+- title: Produkte
+  type: products-group
+  dropdown: products
+  link: /producten.html
```

`url` instead of `link`?

```suggestion
  url: /producten.html
```

---

yep